### ISSUE TITLE
Suggestion: Repeat/Recurring Tasks

### ISSUE THREAD

**Issue by brcmesquita** [alainm23/planify#54]
Imagine that I need to perform a task weekly, monthly or annually. Instead of creating the same task several times, I could create it only once, and mark it to repeat as I need it (daily, weekly, monthly, yearly etc.).

EDIT:
To better exemplify: I want to create a task in Planner, which helps me to remember that I have to pay bills (electricity, water, gas etc). And when I mark that I have completed this month, a new task (in its case the same) is created next month, or next year (for annual reminders).

**Comment by Hedronmx** [comment]
Please add this!! Is the only thing preventing me using this app.

**Comment by nandgator** [comment]
Yes, adding a recurring reminder option will be good. For instance I want to be reminded to complete certain registration before a due date, but I'm waiting for a few documents. So once I get the documents I do not forget to register.

**Comment by bfsgr** [comment]
What's the status on this one? Recurring tasks are indeed a needed feature

**Comment by alainm23** [comment]
@bfsgr  I will be working on this feature for v2.3. However, I think that at the moment this feature can only be used locally and you will not be able to work hand in hand with Todoist.

**Comment by alainm23** [comment]
I'm working on this for v2.3

**Comment by alainm23** [comment]
Working on this in v2.3 and with Todoist support, at the moment only available in English and Spanish language.

**Comment by jaimevillegas** [comment]
Any update on this?
